enable/tests/test_markers.py

```diff
@@ -0,0 +1,52 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import unittest
+
+import numpy as np
+
+from enable.compiled_path import CompiledPath
+from enable.kiva_graphics_context import GraphicsContext
+from enable.markers import CustomMarker
+from enable.tests._testing import skip_if_not_qt
```

Comment:
not an actionable comment - i was wondering if the enable testsuite should always import from `enable.api` instead of importing from the specific modules. Note that this is a more general question for ETS/package development.

Reply:
Yeah I notice some of the test files do but others don't.  I think it probably makes sense to use api everywhere in tests